### ISSUE TITLE
Accessibility bug in Citation Drawer

### ISSUE THREAD

**Issue by resource11** [CLIxIndia-Dev/content_player#19]
When navigating through content player via VoiceOver, the 'close drawer' X button of the drawer is announced on the page. That drawer shouldn't get announced until the citation page is loaded.

We have a solution created in unplatform in both the Activities component and the LessonModal component. See the pattern leveraged there for ideas.

Ideally, the Citation appearanced in announced using an aria-live attribute. Native dialog elements do this already for free if you set them up correctly, and the react-aria-modal library leverages this functionality. 

I'm not sure if a drawer component works the same way, but we need to announce to the user that this component has been loaded to the DOM.

**Comment by coleshaw** [comment]
The X button has the `aria-hidden` attribute added in PR #20 , when it is not visible on the screen. Should address this issue.
